src/glum/_glm.py

```diff
@@ -837,11 +841,20 @@ def _convert_from_pandas(self, df: pd.DataFrame) -> tm.MatrixBase:
         """Convert a pandas data frame to a tabmat matrix."""
         if hasattr(self, "feature_dtypes_"):
             df = _align_df_categories(df, self.feature_dtypes_)
+            if self.cat_missing_method == "convert":
```

Comment:
Here is the logic for when `cat_missing_method` is "convert", but I can't find the logic for handling fail vs zero.

Reply:
Is it implemented in tabmat?

Reply:
Yes, it is implemented in tabmat. Creating a new category for the missings is also implemented in tabmat (e.g. the `fit` method does not need to have this logic), but we have to do it manually in predict-type functions to make sure that we are doing it the same way as at fitting time (e.g. if a categorical column has missing values in the training data, but no missing values in the data used for prediction, we still need to create the missing category for it in the latter).